apiserver/common/model/modelstatus.go

```diff
@@ -156,11 +161,12 @@ func (c *ModelStatusAPI) modelStatus(ctx context.Context, tag string) (params.Mo
 	}
 	modelFilesystems := ModelFilesystemInfo(filesystems)
 
+	// TODO: add life and ownertag values when they are supported in model DB
 	result := params.ModelStatus{
 		ModelTag:           tag,
-		OwnerTag:           model.Owner().String(),
-		Life:               life.Value(model.Life().String()),
-		Type:               string(model.Type()),
+		OwnerTag:           "",
```

Comment:
https://github.com/ycliuhw/juju/blob/dcab486555b5ab7790cac71ebfd9d9a83c3df714/apiserver/facades/client/modelmanager/modelmanager.go#L901
You can get these data from here.

Reply:
We don't have  the ability to correctly get the model owner at the moment from a service that is just model scoped. There is design work a foot to better expose this value. Until this is done can we just place a TODO over it.

Reply:
Thanks for helping to clarify tom, and I have already added a TODO over it, please let me know if there's anything else we need on this

Reply:
I’d prefer to retrieve the information from the controller database for now to avoid breaking existing functionality, and leave a TODO to switch to the model database once the data becomes available there.
But it's ok if we are going to fix the TODO in the near future.

Reply:
I have been told to leave it for the moment by @wallyworld. If we introduce the controller database here we get more problems. This is not the only place we have this TODO.

Reply:
I have set a bad value in this PR. The client will fail at the moment if we don't set a valid tag.